extensions/data-transfer/portability-data-transfer-microsoft/src/test/java/org/dataportabilityproject/transfer/microsoft/transformer/calendar/ToCalendarEventModelTransformerTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2018 The Data-Portability Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataportabilityproject.transfer.microsoft.transformer.calendar;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.dataportabilityproject.transfer.microsoft.helper.TestTransformerContext;
+import org.dataportabilityproject.types.transfer.models.calendar.CalendarAttendeeModel;
+import org.dataportabilityproject.types.transfer.models.calendar.CalendarEventModel;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static org.dataportabilityproject.transfer.microsoft.transformer.TransformConstants.CALENDAR_ID;
+
+public class ToCalendarEventModelTransformerTest {
+    private ToCalendarEventModelTransformer transformer;
+    private ObjectMapper mapper;
+    private TestTransformerContext context;
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testTransform() throws IOException {
+        context.setProperty(CALENDAR_ID, "123");
+        Map<String, Object> rawEvent = mapper.readValue(JSON, Map.class);
+
+        CalendarEventModel event = transformer.apply(rawEvent, context);
+
+        Assert.assertEquals("123", event.getCalendarId());
+
+        Assert.assertEquals("Some Place", event.getLocation());
+        Assert.assertEquals("Test Appointment 1", event.getTitle());
+        Assert.assertTrue(event.getNotes().length() > 5);
+        Assert.assertEquals(1, event.getAttendees().size());
+
+        CalendarAttendeeModel attendee = event.getAttendees().get(0);
+        Assert.assertEquals("Test Test1", attendee.getDisplayName());
+        Assert.assertEquals("foo@foo.com", attendee.getEmail());
+        Assert.assertFalse(attendee.getOptional());
+
+        Assert.assertEquals(18, event.getStartTime().getDateTime().getHour());
+        Assert.assertEquals(0, event.getStartTime().getDateTime().getMinute());
+
+        Assert.assertEquals(18, event.getEndTime().getDateTime().getHour());
+        Assert.assertEquals(30, event.getEndTime().getDateTime().getMinute());
+    }
+
+    @Before
+    public void setUp() {
+        transformer = new ToCalendarEventModelTransformer();
+        mapper = new ObjectMapper();
+        context = new TestTransformerContext();
+    }
+
+    private static final String JSON =
```

Comment:
We can probably use a more descriptive variable name or add comment

Reply:
OK